product.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require_once 'src/Services/ProductsDisplayService.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Factory/furnitureDatabaseConnector.php';
+require_once 'src/Entities/ProductEntity.php';
+
+$db = furnitureDatabaseConnector::connect();
+$product = [];
+$similarProduct = false;
+
+if (!empty($_GET['id']) && is_numeric($_GET['id'])) {
+    $id = $_GET["id"];
+    $product = ProductModel::getIndividualProduct($db, $id);
+}
+if ($product) {
+    $related_id = $product->getRelated();
+    $similarProduct = ProductModel::getSimilarProduct($db, $related_id);
+}
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Furniture Store</title>
+        <script src="https://cdn.tailwindcss.com"></script>
+    </head>
+    <body>
+        <nav class="bg-slate-800 py-2 px-5">
+            <span class="text-4xl text-white">Furniture Store</span>
+        </nav>
+        <header class="container mx-auto md:w-2/3 md:mt-10 py-16 px-8 bg-slate-200 rounded">
+            <?php
+            if ($product){
```

Comment:
```suggestion
            if ($product) {
```

Inconsistent with other usages in this file.